package.json

```diff
@@ -3,17 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@stripe/react-stripe-js": "^1.4.0",
+    "@stripe/stripe-js": "^1.13.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.6.0",
+    "dotenv": "^8.2.0",
     "eslint-config-prettier": "^8.2.0",
     "eslint-plugin-prettier": "^3.3.1",
     "husky": "^6.0.0",
+    "jquery": "^3.6.0",
+    "popper.js": "^1.16.1",
     "prettier": "^2.2.1",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
+    "sweetalert2": "^10.16.2",
+    "web-vitals": "^1.0.1",
+    "eslint": "^7.24.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
Why add eslint here instead of devDependencies? 🤔

Reply:
because when I deployed to heroku I got an error, the solution I found to that error was to remove the Devdependecies

Reply:
Humm, actually that's something that shouldn't go to production, because it will take more time to serve the code because you're just adding another layer, it would be good to do some research to fix this issue within heroku configuration, however, when you're going to deploy a project to some client, it will most likely not using heroku